JavaTraining/BillsBurgers/src/Main.java

```diff
@@ -0,0 +1,63 @@
+public class Main {
+
+    public static void main(String[] args) {
+
+        Burger burger = new Burger("Simple Burger", "white roll", 5);
+        System.out.println(burger.getName() + " in " + burger.getBreadType());
+        burger.setCheese(true);
+        burger.setLettuce(true);
+        burger.setTomato(true);
+//        burger.receipt();
+//        System.out.println();
+        //burger.cheesePrice = 0;
+        burger.receipt();
+        System.out.println();
+
+
+        System.out.println();
+        HealthyBurger healthyBurger = new HealthyBurger("Healthy Burger", 6);
+        System.out.println(healthyBurger.getName() + " in " + healthyBurger.getBreadType());
+        healthyBurger.setTomato(true);
+        healthyBurger.setPeppers(true);
+        healthyBurger.receipt();
+
+        System.out.println();
+
+        DeluxeBurger deluxeBurger = new DeluxeBurger("Deluxe Burger", "seed roll", 5.5);
+        System.out.println(deluxeBurger.getName() + " in " + deluxeBurger.getBreadType());
+        deluxeBurger.receipt();
+        //deluxeBurger.lettuce = true; //QUE PASA SI HAGO ESTO?
+
+
+
+    }
+}
+
+// The purpose of the application is to help a fictitious company called Bills Burgers to manage
```

Comment:
best not to put documetation at the bottom of the file... people expect it at the top (and people scan code very fast... they may never get this far in the file)

Reply:
Also multy line comments go in /*  */ type comments